Cubical/Categories/Displayed/Functor/More.agda

```diff
@@ -171,3 +172,41 @@ module _ {C : Category ℓC ℓC'} {D : Category ℓD ℓD'} {F : Functor C D}
       (Fᴰ .F-homᴰ)
       (Gᴰ .F-homᴰ)
 
+module FunctorᴰNotation {C : Category ℓC ℓC'} {D : Category ℓD ℓD'} {F : Functor C D}
```

Comment:
I think the most sensible thing in light of the below comments would be to just `open Functor (∫F F) public`. Or just use `∫F F` directly, though that's maybe less readable.